src/tastyscrapy/spiders.py

```diff
@@ -48,18 +49,46 @@ def on_login_response(self, response):
         if xjpath.path_lookup(payload, 'status') != ('success', True):
             raise LoginFailedException("Unable to log in using provided credentials.")
 
-        # emit an item which shows that we logged in successfully for audit tracking
-        yield LoginSuccess(created=datetime.utcnow().isoformat())
-
+        # lookup redirect url
         redirect, is_success = xjpath.path_lookup(payload, 'redirect')
 
+        # extract session key for cookie
+        session_key, session_value = xjpath.XJPath(payload)['session'].split('=')
+
         # start parsing items
         yield scrapy.Request(redirect if is_success else "https://del.icio.us/{}".format(
-            self.settings.get('DELICIOUS_USERNAME')), callback=self.on_bookmark_response)
+            self.settings.get('DELICIOUS_USERNAME')), callback=self.on_bookmark_response,
+            cookies = { session_key: session_value })
 
     def on_bookmark_response(self, response):
         """
         Callback method triggered when login has succeeded or the next
         bookmark page has been fetched.
         """
-        pass
+        # get all bookmark blocks on the page
+        for bookmark_block in response.css('.profileMidpanel .articleThumbBlockOuter'):
+            # presumably a md5 of the url or something
+            bookmark_id = bookmark_block.css('::attr(md5)').extract_first()
+            # unix integer in utc timezone
+            bookmark_date = int(bookmark_block.css('::attr(date)').extract_first())
+            bookmark_title = bookmark_block.css('.articleTitlePan a::attr(title)').extract_first()
+            bookmark_url = bookmark_block.css('.articleInfoPan p:first-child a::attr(href)').extract_first()
+            bookmark_comment = bookmark_block.css('.thumbTBriefTxt p::text').extract_first()
+            bookmark_tags = bookmark_block.css('.thumbTBriefTxt .tagName li a::text').extract()
+
+            # convert date from unix integer in utc to an ISO-8601 date string
+            bookmark_date = datetime.utcfromtimestamp(bookmark_date).isoformat()
+
+            # determine whether this is a private link or not
+            bookmark_private = len(bookmark_block.css('.privateArticle')) > 0
+
+            # create an item
+            yield Bookmark(delicious_id=bookmark_id, created=bookmark_date, title=bookmark_title,
+                url=bookmark_url, comment=bookmark_comment, tags=bookmark_tags, private=bookmark_private)
+
+        # after parsing all bookmarks, find the next page and go there
+        next_link = response.css('.pagination a[aria-label=Next]::attr(href)').extract_first()
```

Comment:
As long as this value is found, we keep crawling through the pages one by one by "clicking" the next button.